### PR TITLE
Additional logLevel for customLogger

### PR DESCRIPTION
**Pull request by pb99u069**
Closes # .

### Pre-review checklist

- [x] All code has been formatted using our config (`make format`)
- [ ] Any new API features or modification of existing behavior is covered as defined in our [test plan](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/test-plan.md)
- [ ] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

**Comment by pb99u069** [comment]
this change doesn't need to be noted in the Changelog, I guess?

File: src/Contract/Config.purs [hunk 1]
```diff
@@ -58,6 +58,7 @@ testnetConfig =
   , extraConfig: {}
   , walletSpec: Nothing
   , logLevel: Trace
+  , customLogLevel: Trace
```
Comment (klntsky) [inline src/Contract/Config.purs:61]: We already have log level here, please remove.

File: src/Contract/Monad.purs [hunk 2]
```diff
@@ -244,6 +244,7 @@ type ConfigParams (r :: Row Type) =
   , ctlServerConfig :: Maybe ServerConfig
   , networkId :: NetworkId
   , logLevel :: LogLevel
+  , customLogLevel :: LogLevel
   , walletSpec :: Maybe WalletSpec
   , customLogger :: Maybe (Message -> Aff Unit)
```
Comment (klntsky) [inline src/Contract/Monad.purs:249]:
```suggestion
  , customLogger :: Maybe (LogLevel -> Message -> Aff Unit)
```

It should accept a log level for filtering

**Review by klntsky — CHANGES_REQUESTED** [review]

**Comment by klntsky** [comment]
> this change doesn't need to be noted in the Changelog, I guess?

It changes a type of a public function, so it must be mentioned

**Comment by pb99u069** [comment]
@klntsky I'm struggling with this issue. does what I've done to here make sense in any way? I don't see where the filtering for the customLogger should be implemented

**Pull request closed without merging**